src/changelogs/fetch_changelogs.py

```diff
@@ -68,6 +77,10 @@ def changelogs_to_file(self, output_dir):
         # add any keys that are not yet present or have differing values,
         existing_changelogs.update(self.changelogs)
 
+        # add ones from input-data's changelogs.json, which currently include hero lab changelogs
+        input_changelogs = json_utils.read(os.path.join(input_dir, 'changelogs.json'))
+        existing_changelogs.update(input_changelogs)
```

Comment:
I'm confused what `changelogs.json` is doing. It only contains a single entry for hero-lab. Is there a problem with just reading it like any other raw-changelog file?

Reply:
This herolab entry can't be read like other raw-changelog files because it doesn't have a forum post to read off of, meaning it has no assigned date. After this is merged and we look at integrating it with the manifest-id versioning branch, a cleaner route may present itself, though I doubt it. As then, we won't be trying to assign it a date, but rather assign it a manifest-id or a ClientVersion or something else to attach it to. I'm unsure, but for now the date for herolab changelogs would need to be added this way.

Reply:
Oh the way the git diff showed me the file made it look like there was only a single entry in it, but actually every changelog is represented there, so that's all good